docs/en/02_Developer_Guides/14_Files/05_File_Migration.md

```diff
@@ -100,3 +100,16 @@ SilverStripe\Core\Injector\Injector:
 ```
 
 You can also set this to `0` to disable the limit.
+
+## System Requirements
+
+The approach to running your file migration depends on your system and how many files you are migrating.
+
+Please use the following estimates to decide how you will run your file migration:
+
+| Number of files | Method | Expected Execution Time | Expected Memory Usage |
```

Comment:
```suggestion
| Number of files | Method | Expected Execution Time | Approximate Memory Usage |
```

I think _approximate_ is a better word in this context. I think it's clearer that you will get different results.